RevolveUavcan/Uavcan/UavcanParser.cs

```diff
@@ -27,7 +25,7 @@ public class UavcanParser : IUavcanParser
         /// <param name="logger">Logger used for log output</param>
         /// <param name="dsdlRuleGenerator">DSDL rules to be used in parsing and serialising</param>
         /// <param name="frameStorage">Framestorage that will provide frames to be parsed</param>
-        public UavcanParser(ILogger logger, DsdlRuleGenerator dsdlRuleGenerator, FrameStorage frameStorage)
+        public UavcanParser(ILogger logger, UavcanSerializationRulesGenerator dsdlRuleGenerator, UavcanFrameStorage frameStorage)
```

Comment:
```suggestion
        public UavcanParser(ILogger logger, UavcanSerializationRulesGenerator uavcanSerializationRulesGenerator, UavcanFrameStorage frameStorage)
```
Should change the name everywhere, can use automatic rename in VS